### PR TITLE
Added a basic config to index a mailbox and attachments in Exchange online

### PR DESCRIPTION
**Pull request by aemery**
Did live tests based on our devcoveo tenant. It works at indexing emails and their attachments even for multimedia content.

Next step could be to add an endpoint to discover many mailboxes, for now it crawls only one.

### CODE REVIEW

**Review by mdey1 — APPROVED** [review]
I reviewed it against the Microsoft Graph API documentation, and it lgtm! 👍🏽

File: Exchange Online/ExchangeOnlineSourceConfig.json [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        }
+      },
+      "Endpoints": [
+        {
+          "Method": "GET",
+          "Path": "/messages",
+          "ItemPath": "value",
+          "ItemType": "Email",
+          "QueryParameters": {
+            "$select": "sender,subject,from,toRecipients,body,webLink,sentDateTime,hasAttachments"
```
Comment (ldblanchet) [inline Exchange Online/ExchangeOnlineSourceConfig.json:55]: est-ce toRecipients te retourne les cc et bcc?

**Review by ldblanchet — APPROVED** [review]

File: Exchange Online/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Indexing Azure Active Directory Using the Generic REST API Connector
```
Comment (jlevesquecoveo) [inline Exchange Online/README.md:1]:
```suggestion
# Indexing Exchange Online Using the Generic REST API Connector
```

File: Exchange Online/README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+## Use Case
+This example shows how to a index Exchange Online shared mailbox or a user's mailbox using the Microsoph Graph API.
```
Comment (jlevesquecoveo) [inline Exchange Online/README.md:4]:
```suggestion
This example shows how to index a shared Exchange Online mailbox or a user's mailbox using the Microsoph Graph API.
```

**Review by jlevesquecoveo — APPROVED** [review]

**Pull request merged**